### PR TITLE
Update openSUSE images

### PR DESCRIPTION
**Pull request by flavio**
* Drop openSUSE 13.1.
* Update openSUSE 13.2
* Update openSUSE Tumbleweed

### CODE REVIEW

File: library/opensuse [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-13.1: git://github.com/openSUSE/docker-containers-build@0d21bc58cd26da2a0a59588affc506b977d6a846 docker
-bottle: git://github.com/openSUSE/docker-containers-build@0d21bc58cd26da2a0a59588affc506b977d6a846 docker
-
 # openSUSE 13.2
-13.2: git://github.com/openSUSE/docker-containers-build@cae38d2559dfb9698464bee2e9f4fa97ea017055 docker
-harlequin: git://github.com/openSUSE/docker-containers-build@cae38d2559dfb9698464bee2e9f4fa97ea017055 docker
-latest: git://github.com/openSUSE/docker-containers-build@cae38d2559dfb9698464bee2e9f4fa97ea017055 docker
+13.2: git://github.com/openSUSE/docker-containers-build@b3a02e025a99ecf124be4067e4ac94b2d5cf0ab3 docker
+harlequin: git://github.com/openSUSE/docker-containers-build@b3a02e025a99ecf124be4067e4ac94b2d5cf0ab3 docker
+latest: git://github.com/openSUSE/docker-containers-build@b3a02e025a99ecf124be4067e4ac94b2d5cf0ab3 docker
```
Comment (tianon) [inline library/opensuse]: Shouldn't these three be https://github.com/openSUSE/docker-containers-build/commit/25fa2adb969481d23f71e23ab7ff0148ae4df0ef instead? (`25fa2adb969481d23f71e23ab7ff0148ae4df0ef`)

The commit here is tumbleweed. :+1:

**Comment by flavio** [comment]
@tianon thanks for the comment, I fixed the issue.

**Comment by tianon** [comment]
LGTM :heart:

Build test of #853; 0a3343117a36bec8fe3a33d285e335f0dde28bc4 (`opensuse`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/0a3343117a36bec8fe3a33d285e335f0dde28bc4/library/opensuse"
$ bashbrew build "$url"
Cloning opensuse (git://github.com/openSUSE/docker-containers-build) ...
Processing opensuse:13.2 ...
Processing opensuse:harlequin ...
Processing opensuse:latest ...
Processing opensuse:tumbleweed ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing opensuse:13.2
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...warning: garbage password detected for 'root': '45EM2PQ5Lq.Nw'
passed
testing opensuse:tumbleweed
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...warning: garbage password detected for 'uucp': 'x'
warning: garbage password detected for 'games': 'x'
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'mail': 'x'
warning: garbage password detected for 'ftp': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'root': 'bipIUUGQQVtcc'
warning: garbage password detected for 'wwwrun': 'x'
warning: garbage password detected for 'man': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'news': 'x'
warning: garbage password detected for 'lp': 'x'
passed
```

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**